<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QMainWindow" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>642</width>
    <height>580</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Fira Code</family>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Credit calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>622</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="sum_label">
             <property name="text">
              <string>Sum</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="rate_label">
             <property name="text">
              <string>Rate(%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="term_label">
             <property name="text">
              <string>Term</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="sum_input">
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="minimum">
              <double>1000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100000000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>100.000000000000000</double>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rate_input">
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="maximum">
              <double>10000000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>100.000000000000000</double>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="term_input">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="type_label">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="rate_type_label">
             <property name="text">
              <string>Rate type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="term_type_label">
             <property name="text">
              <string>Term type</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QComboBox" name="type_input">
             <item>
              <property name="text">
               <string>Annuity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Differentiated</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rate_type_input">
             <item>
              <property name="text">
               <string>Per annum</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Monthly</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="term_type_input">
             <item>
              <property name="text">
               <string>Months</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Years</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="calculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="output_table"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionCalc"/>
    <addaction name="actionDeposit"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <action name="actionCalc">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calc</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
